README.md

```diff
@@ -166,3 +166,51 @@ Ownership transferred for TokenMigrator at 0x9615460582Efa2a9b1d8D21e7E02afE43A4
 Ownership transferred for Blocklist at <redacted> to 0xeBc37F4c20C7F8336E81fB3aDf82f6372BEf777E
 Ownership transferred for Allowlist at <redacted> to 0xeBc37F4c20C7F8336E81fB3aDf82f6372BEf777E
 ```
+
+Here's the corrected README section:
+
+---
+
+## Using Development Kit
+
+### Deploy Contracts
+
+You can run the deploy script by specifying tags for the deploy script:
+
+```bash
+yarn deploy:sepolia --tags minter
```

Comment:
hmm looks like running the underlying with tags minter seems to err out 

```
$ npx hardhat deploy --network localhost --tags minter
TypeError: Cannot read properties of undefined (reading 'address') at /04_minter.ts:24:23
```
same with payment splitter. 
wsgeth and sgeth seem to deploy properly and addresses get reused correctly though.

Reply:
It's because deploy script uses second wallet address as mutisig address

https://github.com/chimera-defi/SharedDeposit/blob/3ca0fbb022d509c5c3c1134726655c26ae3c7e35/deploy/03_paymentSplitter.ts#L10

https://github.com/chimera-defi/SharedDeposit/blob/3ca0fbb022d509c5c3c1134726655c26ae3c7e35/hardhat.config.ts#L147

There are two cases can fix this.
- don't use named account for deploy script and use static address(but named accounts provides some benefits for testing and automation).
- we can use mnemonic or can use multiple private keys for deployment in hardhat config

Reply:
hmm, lets do what we used to do in hardhat config and use the default anvil private key for the default address? and make it clear what is expected from the script user to get it to work? 

on mainnet, we wont have the priv key of the multisig as itll be a  gnosis safe, but we can execute txs through it, and ideally we deploy everything first, then transfer anything needed to the multisig

Reply:
yeah. maybe, let me change multiSig address to constant

Reply:
Used constant multisig address. Please check deploy script now